recipes/nvtx/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "nvtx" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nvtx-{{ version }}.tar.gz
+  sha256: ef178af514f6a8a9e6be709b016c0a9951daa5a601beff388bf33382e3f1257f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - cython
+    - pip
+    - python
```

Did we want to add `nvtx-c` as a dependency here?

```suggestion
  host:
    - cython
    - pip
    - python
    - nvtx-c
```

---

Yup! Thanks!